src/Faithlife.Utility/WrappingStream.cs

```diff
@@ -122,12 +122,6 @@ public override int ReadTimeout
 		public override Task<int> ReadAsync(byte[] buffer, int offset, int count, CancellationToken cancellationToken) =>
 			WrappedStream.ReadAsync(buffer, offset, count, cancellationToken);
 
-		/// <summary>
-		/// Asynchronously reads the bytes from the current stream and writes them to another stream, using a specified buffer size and cancellation token.
-		/// </summary>
-		public override Task CopyToAsync(Stream destination, int bufferSize, CancellationToken cancellationToken) =>
-			WrappedStream.CopyToAsync(destination, bufferSize, cancellationToken);
```

Comment:
This simply feels like a bug that the `WrappingStream` class no longer delegates method calls to the stream it wraps.